templates/dartfrog_jwt_auth/routes/me/_middleware.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+import 'package:dart_frog/dart_frog.dart';
+import 'package:dart_jsonwebtoken/dart_jsonwebtoken.dart';
+
+import '../login.dart';
+
+final class AuthData {
+  final String username;
+  const AuthData(this.username);
+}
+
+Handler middleware(Handler handler) {
+  return (context) async {
+    // Retrieve token from header eg: `Bearer xxxxxx` -> `xxxxxx`
+    final bearerToken = context.request.headers[HttpHeaders.authorizationHeader]
```

Comment:
can we have this logic aligned with https://github.com/invertase/globe/pull/73/files#diff-38f5d3106afa19697950a7c86e88f031767c7df053f59ae5f1e64a014ace49d4

Reply:
Resolved.